From 4c55570c74ec990a5798f49de4b283c4644814e1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Oliver=20St=C3=B6neberg?=
 <firewave@users.noreply.github.com>
Date: Thu, 2 Mar 2023 21:50:14 +0100
Subject: [PATCH] cleaned up includes based on `include-what-you-use` (#4599)

MJ: backported only the change for lib/mathlib.cpp which is necessary
    to fix build with gcc-13.

---
Upstream-Status: Backport [https://github.com/danmar/cppcheck/commit/bd1ae69b00fa0be4df7ccea91604707e70129c6c]

 lib/mathlib.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/mathlib.cpp b/lib/mathlib.cpp
index 9457790f8..446b7dc46 100644
--- a/lib/mathlib.cpp
+++ b/lib/mathlib.cpp
@@ -23,6 +23,7 @@
 
 #include <cctype>
 #include <cmath>
+#include <cstdint>
 #include <cstdlib>
 #include <exception>
 #include <limits>
